### PR TITLE
Adding text selection and focus section to the input and label topic

### LINKED ISSUE #1907: [igxFocus] and [igxTextSelection] documentation
Create/update a topic to cover both directives:
- igxFocus - The directive allows you to force focus for every element it is applied
- igxTextSelection - The directive allows you to select the whole text range for every element with text content it is applied.

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #1907

### CODE REVIEW

File: en/components/label_input.md [hunk 1]
```diff
@@ -87,6 +87,46 @@ in our markup:
...
+    <igx-input-group>
+        <input igxInput [igxFocus]="isFocused" name="fullName" type="text" />
+        <label igxLabel for="fullName">Full Name</label>
+    </igx-input-group>
+   ...
+```
+>[!NOTE]
+>To use the [`igxFocus`]({environment:angularApiUrl}/classes/igxfocusdirective.html) directive you have to import the [`IgxFocusModule`]({environment:angularApiUrl}/classes/igxfocusmodule.html).
+
+If you want to select the entire text in `igxInput` element on focus event you have to enable the [`igxTextSelection`]({environment:angularApiUrl}/classes/igxtextselectiondirective.html) directive.
```
Comment (simeonoff) [inline en/components/label_input.md:105]:
```suggestion
If you want the text in an input element, marked with `igxInput`, to be selected on focus, you have to enable the [`igxTextSelection`]({environment:angularApiUrl}/classes/igxtextselectiondirective.html) directive.
```

File: en/components/label_input.md [hunk 2]
```diff
@@ -87,6 +87,46 @@ in our markup:
...
+   ...
+    <igx-input-group>
+        <input igxInput [igxTextSelection]="true" [igxFocus]="isFocused" name="fullName" type="text" />
+        <label igxLabel for="fullName">Full Name</label>
+    </igx-input-group>
+    <igx-input-group>
+        <input igxInput [igxTextSelection]="true" name="email" type="text" />
+        <label igxLabel for="email">Email</label>
+    </igx-input-group>
+   ...
```
Comment (simeonoff) [inline en/components/label_input.md:117]:
```suggestion
<igx-input-group>
    <input igxInput [igxTextSelection]="true" [igxFocus]="isFocused" name="fullName" type="text" />
    <label igxLabel for="fullName">Full Name</label>
</igx-input-group>

<igx-input-group>
    <input igxInput [igxTextSelection]="true" name="email" type="text" />
    <label igxLabel for="email">Email</label>
</igx-input-group>
```

File: en/components/label_input.md [hunk 3]
```diff
@@ -87,6 +87,46 @@ in our markup:
...
+```
+>[!NOTE]
+>To use the [`igxTextSelection`]({environment:angularApiUrl}/classes/igxtextselectiondirective.html) directive you have to import the [`IgxTextSelectionModule`]({environment:angularApiUrl}/classes/igxtextselectionmodule.html).
```
Comment (simeonoff) [inline en/components/label_input.md:120]:
```suggestion
>To use the [`igxTextSelection`]({environment:angularApiUrl}/classes/igxtextselectiondirective.html) directive, you have to import the [`IgxTextSelectionModule`]({environment:angularApiUrl}/classes/igxtextselectionmodule.html).
```

File: en/components/label_input.md [hunk 4]
```diff
@@ -87,6 +87,46 @@ in our markup:
...
+```html
+   ...
+    <igx-input-group>
+        <input igxInput [igxFocus]="isFocused" name="fullName" type="text" />
+        <label igxLabel for="fullName">Full Name</label>
+    </igx-input-group>
+   ...
+```
+>[!NOTE]
+>To use the [`igxFocus`]({environment:angularApiUrl}/classes/igxfocusdirective.html) directive you have to import the [`IgxFocusModule`]({environment:angularApiUrl}/classes/igxfocusmodule.html).
```
Comment (simeonoff) [inline en/components/label_input.md:103]:
```suggestion
>To use the [`igxFocus`]({environment:angularApiUrl}/classes/igxfocusdirective.html) directive, you have to import the [`IgxFocusModule`]({environment:angularApiUrl}/classes/igxfocusmodule.html).
```

**Comment by HristoP96** [comment]
@simeonoff  The requested changes are applied for both en and jp versions. Thank you for your suggestions :).

**Pull request merged**